BootCampV2/Homeworks/SecondWeek/Login.cs

```diff
@@ -0,0 +1,54 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace BootCampV2.Homeworks.SecondWeek
+{
+    class Login
+    {
+        public string Username{ get; private set; }
+        public string Password{ get; private set; }
+
+        public void SetData(string username, string password)
```

Comment:
What happens if you forget to call `SetData`?  Better to pass these values in via the constructor instead of a separate method and then you don't have to worry about this possibility.  Of course, we haven't covered OOP yet :)